compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/DerivedConfigUtils.cpp

```diff
@@ -165,4 +165,31 @@ SmallVector<int64_t> deriveThreadTileSizes(Operation *op) {
       .Default([&](Operation *op) -> SmallVector<int64_t> { return {}; });
 }
 
+static const int64_t kDefaultSubgroupSize = 64; // TODO: can we query this?
```

Comment:
I'm not an expert. Is it something similar to `target.getPreferredSubgroupSize()` where target is a [IREE::GPU::TargetAttr](https://github.com/iree-org/iree/blob/946b1eacf54e591e93c6567b000f4b57c02e7bd1/compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td#L435-L506) attached to executable target?

E.g.,

https://github.com/iree-org/iree/blob/946b1eacf54e591e93c6567b000f4b57c02e7bd1/compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/config_vector_distribute_gfx942.mlir#L86-L93

Reply:
bump this again.

Reply:
Updated!